### PR TITLE
fix anthropogenic total ERF

### PR DESCRIPTION
**Pull request by chrisroadmap**
Accidently set the wrong index for reporting anthropogenic total ERF from FaIR.

- [x] Description in ``CHANGELOG.rst`` added (single line such as: ``(`#XX <https://github.com/openscm/openscm-runner/pull/XX>`_) Added feature which does something``)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
     ("Robert Gieseke", "robert.gieseke@pik-potsdam.de"),
     ("Jared Lewis", "jared.lewis@climate-energy-college.org"),
     ("Sven Willner", "sven.willner@pik-potsdam.de"),
+    ("Chris Smith", "c.j.smith1@leeds.ac.uk")
```
Comment (chrisroadmap) [inline setup.py:14]: I added myself as I wrote the FaIR adapter, but depending on your authorship policies feel free to remove.

**Comment by chrisroadmap** [comment]
possible nan in the anthropogenic. To check.

**Comment by chrisroadmap** [comment]
> possible nan in the anthropogenic. To check.

This wasn't a problem, I was using a dumb setup

**Comment by chrisroadmap** [comment]
Right this should be good to go. Problem summary is that `Effective Radiative Forcing|Anthropogenic` from FaIR was not reporting the right numbers.

**Review by znicholls — APPROVED** [review]
Beauty. If you wanted to be super careful about it you could go and add tests of all radiative forcing outputs now so these errors can be removed with confidence in future (none of the tests needed to change for this update to pass). As you wish.

**Comment by chrisroadmap** [comment]
OK ready. I can squash my last 7 commits where I'm arguing with a computer if you want.

File: src/openscm_runner/adapters/fair_adapter/_run_fair.py [hunk 2]
```diff
@@ -210,17 +210,16 @@ def _process_output(fair_output, output_vars, factors):  # pylint: disable=R0915
...
     )
     # This definition does not include ozone and H2O from CH4 oxidation
-    data["Effective Radiative Forcing|Greenhouse Gases|Kyoto Gases"] = np.sum(
-        forcing[:, :15], axis=1
-    )
+    data["Effective Radiative Forcing|Kyoto Gases"] = np.sum(forcing[:, :15], axis=1)
     data["Effective Radiative Forcing|CO2, CH4 and N2O"] = np.sum(
         forcing[:, :3], axis=1
     )
+    # What is the rigorous definition here? CFCs are not included but contain F
```
Comment (znicholls) [inline src/openscm_runner/adapters/fair_adapter/_run_fair.py:222]: This is an excellent question, I'm guessing I followed either a) WG3 or b) one of the assessment report tables but I agree this should be sorted out in RCMIP somewhere

**Review by znicholls — APPROVED** [review]
